packages/flutter_adaptive_scaffold/lib/src/breakpoints.dart

```diff
@@ -30,51 +30,154 @@ class Breakpoints {
   static const Breakpoint standard = WidthPlatformBreakpoint(begin: -1);
 
   /// A window whose width is less than 600 dp and greater than 0 dp.
-  static const Breakpoint small = WidthPlatformBreakpoint(begin: 0, end: 600);
+  static const Breakpoint small = WidthPlatformBreakpoint(
+    begin: 0,
+    end: 600,
+    bottom: 480,
+  );
 
   /// A window whose width is greater than 0 dp.
-  static const Breakpoint smallAndUp = WidthPlatformBreakpoint(begin: 0);
+  static const Breakpoint smallAndUp = WidthPlatformBreakpoint(
+    begin: 0,
+    bottom: 480,
+  );
 
   /// A desktop screen whose width is less than 600 dp and greater than 0 dp.
-  static const Breakpoint smallDesktop =
-      WidthPlatformBreakpoint(begin: 0, end: 600, platform: _desktop);
+  static const Breakpoint smallDesktop = WidthPlatformBreakpoint(
+    begin: 0,
+    end: 600,
+    bottom: 480,
+    platform: _desktop,
+  );
 
   /// A mobile screen whose width is less than 600 dp and greater than 0 dp.
-  static const Breakpoint smallMobile =
-      WidthPlatformBreakpoint(begin: 0, end: 600, platform: _mobile);
+  static const Breakpoint smallMobile = WidthPlatformBreakpoint(
+    begin: 0,
+    end: 600,
+    bottom: 480,
+    platform: _mobile,
+  );
 
   /// A window whose width is between 600 dp and 840 dp.
-  static const Breakpoint medium =
-      WidthPlatformBreakpoint(begin: 600, end: 840);
+  static const Breakpoint medium = WidthPlatformBreakpoint(
+    begin: 600,
+    end: 840,
+    top: 480,
+    bottom: 900,
+  );
 
   /// A window whose width is greater than 600 dp.
-  static const Breakpoint mediumAndUp = WidthPlatformBreakpoint(begin: 600);
+  static const Breakpoint mediumAndUp = WidthPlatformBreakpoint(
+    begin: 600,
+    top: 480,
+    bottom: 900,
+  );
 
   /// A desktop window whose width is between 600 dp and 840 dp.
-  static const Breakpoint mediumDesktop =
-      WidthPlatformBreakpoint(begin: 600, end: 840, platform: _desktop);
+  static const Breakpoint mediumDesktop = WidthPlatformBreakpoint(
+    begin: 600,
+    end: 840,
+    top: 480,
+    bottom: 900,
+    platform: _desktop,
+  );
 
   /// A mobile window whose width is between 600 dp and 840 dp.
-  static const Breakpoint mediumMobile =
-      WidthPlatformBreakpoint(begin: 600, end: 840, platform: _mobile);
+  static const Breakpoint mediumMobile = WidthPlatformBreakpoint(
+    begin: 600,
+    end: 840,
+    top: 480,
+    bottom: 900,
+    platform: _mobile,
+  );
+
+  /// A window whose width is between 840 dp and 1200 dp.
+  static const Breakpoint expanded = WidthPlatformBreakpoint(
```

Comment:
Would `mediumLarge` be a better name?  Expanded makes me think of the `Expanded` widget, which would imply different mechanics for this one, and I don't think that's the case, it's just "medium large", right?

Reply:
@gspencergoog I guess that's fine, but i just wanted to keep it as close to the material naming as possible which would be `compact` for small and `expanded` for mediumLarge.

I'll move to mediumLarge to align with the current code base.